src/main/java/nl/avans/drivio/model/HydrogenCar.java

```diff
@@ -1,13 +1,15 @@
 package nl.avans.drivio.model;
 
+import java.util.Collection;

```

Comment:
Dit hele bestand is anders in mijn code, ik zou deze niet gebruiken en mijn code hiervoor gebruiken